cfme/tests/automate/__init__.py

```diff
@@ -65,3 +67,23 @@ def fill_dialog_field(list)
 end
         """
 )
+
+tag_values = [fauxfactory.gen_alphanumeric(start="tag_", length=12).lower(), "{tag_exist}"]
+tag_delete_from_category = (
+    f"""
+    # Create tag under category - 'Location'
+    $evm.execute(
+'tag_create', 'location', :name => '{tag_values[0]}', :description => 'Testing tag {tag_values[0]}'
+)
+    # Check if tag exists
+    tag_exist = $evm.execute('tag_exists?', 'location', '{tag_values[0]}')
+    $evm.log(:info, "Tag exists: #{tag_values[1]}")
+
+    # Delete the tag
+    $evm.execute('tag_delete', 'location', '{tag_values[0]}')
+
+    # Check if deleted tag exists
+    tag_exist = $evm.execute('tag_exists?', 'location', '{tag_values[0]}')
+    $evm.log(:info, "Tag exists: #{tag_values[1]}")
```

Comment:
**Question/Suggestion**
@ganeshhubale everywhere you use `{tag_vaules[0]}`, why here `{tag_values[1]}`, is there any specific reason.? If any could please add more information around it.

Reply:
No need of more info. It's just list of two values used in ruby code: ```tag_name``` and ```ruby_variable```(So that formatting with fstring will be easy)

Reply:
Okay.